src/components/checkbox/checkbox.scss

```diff
@@ -0,0 +1,401 @@
+@import "default-theme";
+@import "theme-functions";
+@import "variables";
+
+/** The width/height of the checkbox element. */
+$md-checkbox-size: 18px !default;
+/** The width of the line used to draw the checkmark / mixedmark. */
+$md-checkbox-mark-stroke-size: 2/15 * $md-checkbox-size !default;
+/** The width of the checkbox border shown when the checkbox is unchecked. */
+$md-checkbox-border-width: 2px;
+/** The color of the checkbox border. */
+$md-checkbox-border-color: if($md-is-dark-theme, rgba(white, 0.7), rgba(black, 0.54)) !default;
```

Comment:
The spec [explicitly specifies](https://www.google.com/design/spec/components/selection-controls.html#selection-controls-checkbox) the colors that the checkbox border should be: 

> Light
>
> On: Swatch 500, Opacity  100%
> Off: #000000, Opacity  54%
> Disabled: #000000, Opacity  26%

> Dark
>
> On: Swatch 500, Opacity  100%
> Off: #FFFFFF, Opacity  70%
> Disabled: #FFFFFF, Opacity  30%